api/src/main/resources/config/application-dev.properties

```diff
@@ -13,6 +13,9 @@ management.endpoint.health.show-details=always
 
 spring.flyway.enabled=true
 
+es.username=
```

Comment:
The local Docker stack will need with a preconfigured username and password. Trace through using ELASTICSEARCH_URI, and follow this pattern.